src/Reflection/SignatureMap/functionMap.php

```diff
@@ -7178,7 +7178,7 @@
 'mysqli::next_result' => ['bool'],
 'mysqli::options' => ['bool', 'option'=>'int', 'value'=>'mixed'],
 'mysqli::ping' => ['bool'],
-'mysqli::poll' => ['int|false', 'read'=>'array', 'error'=>'array', 'reject'=>'array', 'sec'=>'int', 'usec='=>'int'],
+'mysqli::poll' => ['int|false', '&read'=>'array', '&error'=>'array', '&reject'=>'array', 'sec'=>'int', 'usec='=>'int'],
```

Comment:
I don't think this is valid syntax. It has to be `&rw_read` or `&w_read`.

Reply:
By reading the header of this file I thought the prefix was not required.
Anyway it's fixed now.